hydra_python_core/doc_writer.py

```diff
@@ -467,11 +465,13 @@ def __init__(self, collection: HydraCollection) -> None:
         self.name = collection.name
         self.supportedOperation = collection.supportedOperation
         self.manages = collection.manages
-        if collection.path:
+        if collection.path: 
             self.id_ = "{}EntryPoint/{}".format(DocUrl.doc_url, quote(collection.path, safe=''))
+            self.base_url = DocUrl.doc_url
         else:
-            self.id_ = "{}EntryPoint/{}".format(DocUrl.doc_url, quote(self.name, safe=''))
-
+            self.id_ = "{}EntryPoint/{}".format(DocUrl.doc_url, quote(collection.path, safe=''))
```

Comment:
Just a minor typo. If `collection.path` isn't available , `self.name` should be used.